services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:

```

Comment:
Invalid type hint for return value. It returns `QuerySet[CinemaHall]`
```suggestion
def get_cinema_halls() -> QuerySet[CinemaHall]:

```